wordlist.txt

```diff
@@ -315,3 +315,7 @@ dApp
 leaderboards
 ReferralSet
 CreateReferralSet
+enum
```

Comment:
**thought (non-blocking)**: The spoell checker already ignores things in backtick (`) codeblocks. It looks like these are required because $this style$ is... latex or something? Anyway. I wonder if we can configure the spellchecker to ignore lines with those in... shrug 

Non-blocking because this works fine too.

Reply:
Yeah exactly, anything in $$ is latex and doesn't get caught by the spellchecked. Definitely agree we should look into a more elegant solution then dumping everything in the wordlist when we have some time.